src/components/InputTime/AsString.tsx

```diff
@@ -65,6 +66,34 @@ const AsString: React.FC<AsStringProps> = ({
   // value is the true, controlled value, patched for undefined / empty
   const value = useMemo(() => valueOrUndefined || '', [valueOrUndefined]);
 
+  const step: number | undefined = useMemo(() => {
+    let safeStep: number | undefined;
+    safeStep = typeof customStep === 'number' ? customStep : safeStep;
+
+    safeStep =
```

Comment:
Native HTML input `step` attributes are numbers as strings, so React's built-in Input types allow both.
I'm playing nice by parsing both.